lib/vfscore/automount.c

```diff
@@ -900,5 +906,10 @@ err_umount: __maybe_unused
 	vfscore_autoumount(NULL);
 	return rc;
 }
+#endif /* CONFIG_LIBVFSCORE_AUTOMOUNT */
 
+#if CONFIG_LIBVFSCORE_AUTOMOUNT && CONFIG_LIBVFSCORE_AUTOUNMOUNT
 uk_rootfs_initcall_prio(vfscore_automount, vfscore_autoumount, 4);
+#elif CONFIG_LIBVFSCORE_AUTOUNMOUNT
+uk_rootfs_initcall_prio(0x0, vfscore_autoumount, 4);
+#endif
```

Comment:
Forgot the `/* !CONFIG_LIBVFSCORE_AUTOMOUNT && !CONFIG_LIBVFSCORE_AUTOUNMOUNT */` :stuck_out_tongue_closed_eyes: